### PR TITLE
#9285 Save current group in scenario list window

### PR DESCRIPTION
**Pull request by Spongeloaf**
[#9285 Save current group in scenario list window](https://github.com/OpenRCT2/OpenRCT2/issues/9285)

There is still some work to do before this is ready, but I wanted to ensure I knew how to rebase from upstream first. 

Also, I have some artifacts from visual studio in here. I'm not sure why directories.txt was changed. `openrct2-dll.vcxproj`  was only changed to troubleshoot some linker error related to my initial work. Should I just checkout copies of the original files?

Also, Also, is it normal to see all of my old commits? I though that rebasing would squash them.

### CODE REVIEW

**Comment by duncanspumpkin** [comment]
First of all make your changes on a seperate branch. Don't make them on develop. After you have them on a new branch you can then clean up your develop by reseting it back to upstream. Then you can rebase your branch to remove the bad commits.

File: src/openrct2/directories.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
```
Comment (AaronVanGeffen) [inline src/openrct2/directories.txt]: Please remove this file.

File: src/openrct2-dll/openrct2-dll.vcxproj [hunk 2]
```diff
@@ -46,6 +46,7 @@
     <Link>
       <AdditionalDependencies>libopenrct2.lib;libopenrct2ui.lib;%(AdditionalDependencies)</AdditionalDependencies>
       <ProgramDatabaseFile>$(OutDir)openrct2-dll.pdb</ProgramDatabaseFile>
+      <ShowProgress Condition="'$(Configuration)|$(Platform)'=='Release|x64'">LinkVerbose</ShowProgress>
```
Comment (AaronVanGeffen) [inline src/openrct2-dll/openrct2-dll.vcxproj]: Please undo this change.

**Comment by AaronVanGeffen** [comment]
Thanks for tackling this. It should make for a nice feature — I'm sure it will be appreciated by many players.

I have yet to have a proper look, but as Duncan already pointed out, this will need to be rebased. Please also undo the changes to the VS project and remove the empty World.cpp file.

**Comment by Spongeloaf** [comment]
So, resetting my fork to upstream force-closed the PR. I can't change the branch the PR was based on, or at least I can't figure out how to do it. Should I just make a new PR? Or can this on be changed to my new branch [9285_scenario_group_tab](https://github.com/Spongeloaf/OpenRCT2/tree/feature/9285_scenario_group_tab)

**Comment by duncanspumpkin** [comment]
I'm not 100% sure if it's possible. Just open another pr since we haven't got round to reviewing this properly.

**Comment by Spongeloaf** [comment]
OK thanks. Sorry for the hassle, I'm still very new to git. And thanks for the assistance.
